components/ui/fab/index.tsx

```diff
@@ -117,7 +117,7 @@ type IFabProps = Omit<React.ComponentPropsWithoutRef<typeof UIFab>, 'context'> &
 
 const Fab = React.forwardRef<React.ComponentRef<typeof UIFab>, IFabProps>(
   function Fab(
-    { size = 'md', placement = 'bottom right', className, ...props },
+    { size = 'md', placement = 'bottom right', className, testID,...props },
```

Comment:
Great! 

I'm late to the party, but this should be contributed to the main gluestack repo.